output.js

```diff
@@ -229,6 +229,11 @@ function color(config, colorName, str) {
         return str;
     }
 
+    // Labels like "FAILED" or "PASSED" need a bit of visual padding.
+    if (['FAILED', 'PASSED'].includes(str)) {
```

Comment:
Shouldn't this look at `startsWith`? For example, a stack trace could include `FAILED`

Reply:
I think we don't want to pad just any string that may start with `FAILED`. We only want to do that for the status line.